telemetry telemetry_request
RequestType request_type
---
telemetry telemetry_response